app/Document/Document.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Document;
+
+use App\Document\Paragraph;
+
+class Document
+{
+    public $content;
+
+    public function __construct($filePath)
+    {
+        $this->content = file_get_contents($filePath);
+    }
+
+    public function paragraph($number = null){
```

Comment:
I don't really understand this api.  So you can call a method on a document as in `$document->paragraph(3)` which will return a `Paragraph` object. The `Paragraph` object has been passed the entire document but you can call the plural method `$document->paragraph(3)->paragraphs()` on it to get a string output of the 3rd paragraph.

Or you can call the sentence method on it: `$document->paragraph(3)->sentence(2)` to get a `Sentence` object which can return a string of the specified sentence by calling: `$document->paragraph(3)->sentence(2)->sentences()`

How do you envision this being used by the system?